cognite/neat/_session/_show.py

```diff
@@ -14,26 +18,147 @@ class ShowAPI:
     def __init__(self, state: SessionState) -> None:
         self._state = state
         self.data_model = ShowDataModelAPI(self._state)
+        self.instances = ShowInstanceAPI(self._state)
+
+
+class ShowInstanceAPI:
+    def __init__(self, state: SessionState) -> None:
+        self._state = state
+
+    def __call__(self) -> Any:
+        if not self._state.store.graph:
+            print("No graph data available.")
+            return None
+
+        widget = CytoscapeWidget()
+        widget.layout.height = "700px"
+
+        G, types = self._get_instance_di_graph_and_types()
```

Comment:
G = networkx Graph?

Reply:
renamed it to NxGraph so we know it is Networkx Graph